CHANGELOG.md

```diff
@@ -5,6 +5,14 @@
 - [Amazon Linux 2 release notes](https://docs.aws.amazon.com/AL2/latest/relnotes/relnotes-al2.html)
 - [Amazon Linux AMI 2018.03 Release Notes](https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/)
 
+## 20240905
```

Comment:
Q: Should we also include the new runc version? (e.g. https://github.com/aws/amazon-ecs-ami/blame/6d34b7442aec792eb4cd87a80077e513d266b5d8/CHANGELOG.md#L278-L279)

Reply:
Er I guess it will also be updated in the release notes.

Reply:
If strictly following the runbook, only the following are included, if applicable:
- ecs version
- al1 ami version
- source al1 ami
- al2 ami version
- source al2 ami
- source al2 arm ami
- source al2 kernel 5.10 ami
- source al2 kernel 5.10 arm ami
- al2023 ami version
- source al2023 ami
- source al2023 arm ami
- distribution al2023 release
- feature
- enhancement
- bug fix